src/Mod/AddonManager/addonmanager_workers.py

```diff
@@ -130,6 +140,27 @@ def run(self):
             for repo in repos:
                 self.addon_repo.emit(repo)
             self.info_label.emit(translate("AddonsInstaller", "Workbenches list was updated."))
+        git_exe = utils.checkGitBinary()
+        import platform
```

Comment:
This is already mentioned on line 323  
https://github.com/FreeCAD/FreeCAD/pull/3420/files#diff-86151fd926d4afd8c19a364370c576f0R323

Reply:
If I comment out either of the import platform lines then errors are raised. I have noticed that I can remove a couple of import re statements which I will correct.